plugins/dell-dock/fu-plugin-dell-dock.c

```diff
@@ -165,6 +188,16 @@ void
 fu_plugin_device_registered (FuPlugin *plugin, FuDevice *device)
 {
 	/* thunderbolt plugin */
+	if (g_strcmp0 (fu_device_get_plugin (device), "thunderbolt") == 0 &&
+	    fu_device_has_guid (device, "TBT-00d4b071")) {
+		fu_device_inhibit (device, "dell-dock-usb4", "handled by usb");
```

Comment:
assuming this approach works, this description might need to be elaborated since this can end up at and end user seeing it.

Reply:
ok, will refine the context.